openquake/commonlib/export/hazard.py

```diff
@@ -126,6 +126,36 @@ def export_ruptures_xml(ekey, dstore):
     return [dest]
 
 
+@export.add(('rup_data', 'csv'))
```

Comment:
Good point. There is some confusion here, since this is essentially a view over `rup_data` and `events` that becomes a `ses` output. I am not clear myself of what the name should be. Let's keep `rup_data` for the moment but it could change in the future.